Workade/Views&ViewModels/Explore/ExploreViewController.swift

```diff
@@ -0,0 +1,249 @@
+//
+//  ExploreViewController.swift
+//  Workade
+//
+//  Created by 김예훈 on 2022/11/21.
+//
+
+import UIKit
+import SafariServices
+
+class ExploreViewController: UIViewController {
```

Comment:
[P1] 더 이상 상속을 하지 않는다면 final로 선언 해주세요.
```suggestion
final class ExploreViewController: UIViewController {
```

Reply:
수정했습니다!!